wren-core-py/tests/test_modeling_core.py

```diff
@@ -193,6 +194,7 @@ def test_extract_by(dataset, expected_models):
     extracted_manifest = ManifestExtractor(manifest_str).extract_by(dataset)
     assert len(extracted_manifest.models) == len(expected_models)
     assert [m.name for m in extracted_manifest.models] == expected_models
+    assert extracted_manifest.data_source.__str__() == "DataSource.BigQuery"
```

Comment:
It changes the `data_source` type. Could we keep the field as a string when deserding in the core?

Reply:
> It changes the data_source type
>
What did you mean by this? I think it's still an enum but invoking to_string here.

Reply:
I mean it should be a string, not an enum. Because the `data_source` in the source manifest is a string.
And I am not sure the enum from the core could equal the `app.model.data_source.DataSource`.

Reply:
>I mean it should be a string, not an enum. Because the data_source in the source manifest is a string. 
>

`extracted_manifest` is a python class. Its `data_source` is an enum. The `data_source` will never be a string after deserialized from a JSON to the class.

> And I am not sure the enum from the core could equal the app.model.data_source.DataSource
>
Here is just a way to assert the data source to still be `BigQuery` after extraction. It's not related to the JSON serialize and deserialize.